### DISCUSSION TITLE
Feature support merge 2 .mega-linter.yml configs into one file

### DISCUSSION THREAD

**Discussion by karl18** [oxsecurity/megalinter#3105] [Ideas]
Hi, 
As per our MegaLinter implementation in our org. We try to set two `.megalinter.yml` configuration files and merge them into one file to enable linting. I see - as per the checked-out repo I define in the pipeline, either ONLY on configuration file possible to be used.

The idea here is to have two files:
1. A **base** conf file `.megalinter.yml` for DevOps Team to all our repos which we define in the base PR pipeline where all developers use
2. A **developer** conf file `.megalinter.yml`on every **development team's** repo, where they define in their repo with desired linters.

Means I want to merge two `.megalinter.yml` yaml files during the pipeline process and use the merged file to be used with MegaLinter.

Is this possible? If not, how this implementation is possible with current MegaLinter conf?

I can paste the configuration, which I try for this if needed 
MegaLinter Version: 7.5
CICD: Azure Pipelines \ DevOps

**Comment by karl18** [comment]
I appreciate to get any replies here :) thx a million!

**Comment by nvuillam** [comment]
@karl18 you can store a base MegaLinter file in a public url (like a GitHub repo) and in your repositories, use EXTENDS property :)


https://megalinter.io/latest/config-variables/

Example here: https://github.com/hardisgroupcom/sfdx-hardis/blob/main/defaults/ci/.mega-linter.yml
